python/requirements.txt

```diff
@@ -53,7 +53,7 @@ scikit-image
 prometheus_client>=0.7.1
 requests
 pandas
-tensorboardX>=1.9
+tensorboardX<=2.6.0,>=1.9
```

Comment:
can you add a comment why we need to pin? so that we can easily get the context for future upgrading.

Reply:
done.